### PR TITLE
Add custom key/value sample for Store Sales upload

### PR DESCRIPTION
**Pull request by jim-brook**
Updated Remarketing/UploadStoreSalesTransactions.cs to support custom key and custom value

### CODE REVIEW

**Comment by jradcliff** [comment]
@AnashOommen , @AndrewMBurke , please hold off on reviewing or approving this example until we wrap up the review of the golden example in Java:

https://github.com/googleads/google-ads-java/pull/316

Thanks

File: examples/Remarketing/UploadStoreSalesTransactions.cs [hunk 1]
```diff
@@ -1,490 +1,501 @@
...
+        ///     omit this parameter.</param>
+        /// <param name="externalId">Optional (but recommended) external ID to identify the offline
+        ///     user data job.</param>
+        /// <param name="advertiserUploadDateTime">Date and time the advertiser uploaded data to the
+        ///     partner. Only required if uploading third party data.</param>
+        /// <param name="bridgeMapVersionId">Version of partner IDs to be used for uploads. Only
+        ///     required if uploading third party data.</param>
+        /// <param name="partnerId">ID of the third party partner. Only required if uploading third
+        ///     party data.</param>
+        /// <param name="customKey">Optional: If uploading data with custom key and values, also specify this value.</param>
```
Comment (AndrewMBurke) [inline examples/Remarketing/UploadStoreSalesTransactions.cs:187]: Rephrase for consistency with other descriptions, perhaps "Custom key name. Only required if uploading data with custom key and values."

File: examples/Remarketing/UploadStoreSalesTransactions.cs [hunk 2]
```diff
@@ -1,490 +1,501 @@
...
+                        AddressInfo = new OfflineUserAddressInfo()
+                        {
+                            State = "NY"
+                        }
+                    },
+                },
+                TransactionAttribute = new TransactionAttribute()
+                {
+                    // OPTIONAL: If uploading data with custom key and values, also specify the following value:
+                    // CustomValue = "Painting",
```
Comment (AndrewMBurke) [inline examples/Remarketing/UploadStoreSalesTransactions.cs:362]: Please move this to the bottom of the block; the other values are required.

**Comment by AnashOommen** [comment]
This was merged on https://github.com/googleads/google-ads-dotnet/pull/233

**Pull request closed without merging**